modules/cli/src/main/scala/coursier/cli/params/DependencyParams.scala

```diff
@@ -57,6 +58,23 @@ object DependencyParams {
             )
       }
 
+    val fromFilesDependenciesV: ValidatedNel[String, Seq[String]] =
+      if (options.dependencyFile.isEmpty)
+        Validated.validNel(Seq.empty)
+      else {
+        val files = options.dependencyFile
+        val fromFileDependencies = files.flatMap { file =>
+          val source = Source.fromFile(file)
+          val lines =
+            try source.mkString.split("\n").filter(_.nonEmpty)
```

Comment:
Maybe we should also allow separating dependencies with whitespace or e.g. `,` ?